modules/geochemistry/unit/Makefile

```diff
@@ -12,7 +12,6 @@ THIS_MODULE_DIR    ?= $(shell dirname `pwd`)
 MODULE_DIR         ?= $(shell dirname $(THIS_MODULE_DIR))
 MOOSE_DIR          ?= $(shell dirname $(MODULE_DIR))
 FRAMEWORK_DIR      ?= $(MOOSE_DIR)/framework
-ADDITIONAL_CPPFLAGS += -Wall -Wextra
```

Comment:
@permcody I'm not sure I like removing the extra warning flags here, but the `SampleTest.C` unit test fails our module testing with it added due to an unused value. These flags *are* added in every module that I've looked at (which is why it's here in the first place), but not by default in the Apps. My question for you - is it better to remove these flags, or adjust the SampleTest?

Reply:
So these added flags are a bit redundant in the context I was referring to, as the Pthreads+OpenMP and Modules Debug recipes adds them  anyway. I will remove this commit for consistency with the other modules. Since @WilkAndy is adding his own unit tests, and this example will be removed eventually, perhaps `SampleTest.C:22` should be removed as well, just to push this through the final testing hurdle.